index.html

```diff
@@ -182,6 +267,7 @@ <h3>Contact me</h3>
           </form>
           <div class="bar"><img src="image/Shape.png" alt="Black horizontal line"></div>
         </div>
+       </div>
       </section>
```

Comment:
- Kindly add a `background-image` for the contact section in the desktop version to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155533757-46bd1a9b-8b47-4b81-a7b6-e848df4b23ef.png) | ![Template 2-Main (2) - Copy](https://user-images.githubusercontent.com/24703171/155533866-8f65b024-6de5-44a3-98af-8ffd4cf9b2fe.png) |

- Kindly change the wrapper tag for the contact section to be `footer` instead of `section` for semantic purposes.